Editor/Bootstrap/RegionBootstrap.cs

```diff
@@ -11,7 +11,7 @@ namespace AmazonGameLift.Editor
     /// <summary>
     /// Keeps the region UI state and saves it to settings.
     /// </summary>
-    internal class RegionBootstrap
+    public class RegionBootstrap
```

Comment:
Why does this need to be `public`? Isn't this accessed only by the Editor?

Reply:
Updated